docs/root/configuration/http_conn_man/headers.rst

```diff
@@ -474,7 +474,7 @@ Custom request/response headers can be added to a request/response at the weight
 route, virtual host, and/or global route configuration level. See the
 :ref:`v2 <envoy_api_msg_RouteConfiguration>` API documentation.
 
-No *:*-prefixed pseudo-header may be modified via this mechanism. The *:path*
+No *:\*-prefixed pseudo-header* may be modified via this mechanism. The *:path*
```

Comment:
The intention was just for consistency since the other header examples are italicized. It also appears the intention was to do that as well but there was a trailing asterisk left off. I can remove the italics. What do you think @mattklein123 ?

Reply:
Personally I would either do:

:-prefixed pseudo header (no italics)

or 

*:-prefixed* pseudo header (italics just for the header part)